content/publish/form.json

```diff
@@ -138,7 +138,72 @@
             "computeHelp": "For a compute dataset, your file should match the file type required by the algorithm, and should not exceed 1 GB in file size. ",
             "prominentHelp": true,
             "type": "files",
-            "required": true
+            "required": true,
+            "methods": true,
+            "innerFields": [
+              {
+                "value": "headers",
+                "title": "Headers",
+                "label": "Headers",
+                "placeholder_value": "Authorization",
+                "help": "This QUERY will be stored encrypted after publishing.",
+                "type": "headers",
+                "required": false
+              }
+            ]
+          },
+          {
+            "value": "graphql",
+            "title": "Graphql",
+            "label": "URL",
+            "placeholder": "e.g. http://172.15.0.15:8000/subgraphs/name/oceanprotocol/ocean-subgraph",
+            "help": "This URL will be stored encrypted after publishing. **Please make sure that the endpoint is accessible over the internet and is not protected by a firewall or by credentials.**",
+            "computeHelp": "For a compute dataset, your file should match the file type required by the algorithm, and should not exceed 1 GB in file size. ",
+            "prominentHelp": true,
+            "type": "files",
+            "required": true,
+            "innerFields": [
+              {
+                "value": "headers",
+                "title": "Headers",
+                "label": "Headers",
+                "placeholder_value": "Authorization",
+                "help": "This HEADER will be stored encrypted after publishing.",
+                "type": "headers",
+                "required": false
+              },
+              {
+                "value": "query",
+                "title": "Query",
+                "label": "Query",
+                "placeholder": "query{\n    nfts(\n       orderBy: createdTimestamp,\n       orderDirection:desc\n   ){\n       id\n       symbol\n       createdTimestamp\n   }\n}",
+                "help": "This QUERY will be stored encrypted after publishing.",
+                "type": "codeeditor",
+                "required": true
+              }
+            ]
+          },
+          {
+            "value": "smartcontract", 
+            "title": "Smartcontract",
+            "label": "Address",
+            "placeholder": "e.g. 0x8149276f275EEFAc110D74AFE8AFECEaeC7d1593",
+            "help": "This ABI will be stored encrypted after publishing.",
```

Comment:
same as on editMetadata.json, why are there multiple `help` keys?

Reply:
fixed 👍🏽 yeah, they are used by the same component we have in publish.

<img width="1118" alt="Screenshot 2023-02-09 at 08 17 38" src="https://user-images.githubusercontent.com/13741335/217810921-943b2400-5ccb-40b2-8e01-100e147a3db9.png">